src/traces/isosurface/iso_defaults.js

```diff
@@ -0,0 +1,100 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Registry = require('../../registry');
+var colorscaleDefaults = require('../../components/colorscale/defaults');
+
+module.exports = function supplyIsoDefaults(traceIn, traceOut, defaultColor, layout, coerce) {
```

Comment:
Nicely done! But I would prefer having this `supplyIsoDefaults` in `isosurface/defaults.js`. 

We're trying to move away form this pattern where each defaults-related functions would get their own files. We now prefer having just one `defaults.js` file per trace module and have its reusable pieces exported along with the main trace `supplyDefaults.` function. See [`box/defaults.js`](https://github.com/plotly/plotly.js/blob/master/src/traces/box/defaults.js) for an example.

Reply:
Done in https://github.com/plotly/plotly.js/pull/3488/commits/af17adc5a50ecae9d3d063241b2b94f8ef2e9bac.